src/Utilities/Compiler/Lightup/LightupHelpers.cs

```diff
@@ -10,58 +11,21 @@ namespace Analyzer.Utilities.Lightup
 {
     internal static class LightupHelpers
     {
-        internal static Func<TSyntax, TProperty> CreateSyntaxPropertyAccessor<TSyntax, TProperty>(Type type, string propertyName)
+        internal static Func<TSyntax, TProperty> CreateSyntaxPropertyAccessor<TSyntax, TProperty>(Type? type, string propertyName)
```

Comment:
Could we avoid the suppression on `default` in `FallbackAccessor` by constraining `TProperty` with `class` and annotating `Func<TSyntax, TProperty?>`?

Reply:
We could, but then it's not as useful.

Reply:
I'll update it in a separate pull request to explicitly specify the fallback return value.

Reply:
➡️ #3374